src/components/ModalGal/ModalGal.jsx

```diff
@@ -1,9 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import CurrentPhoto from '../CurrentPhoto/CurrentPhoto';
+import PhotoList from '../PhotoList/PhotoList';
+import { X, Left, Right } from '../SVG/SVG';
+import styles from './ModalGal.css';
 
-const ModalGal = (props) => {
-  return (
-    <div> ModalGal Component </div>
-  );
+
+class ModalGal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      photosInfo: this.props.photosInfo,
+      currentPhoto: this.props.photosInfo[0].uri,
+    };
+  }
+
+  renderModal() {
+    return (
+      <div className={styles.container}>
+        <div className={styles['close-control-bar']}>
+          <X />
+        </div>
+        <Left />
+        <Right />
+        <CurrentPhoto currentPhoto={this.state.currentPhoto} />
+        <PhotoList photosInfo={this.state.photosInfo} />
+      </div>
+    );
+  }
+
+  render() {
+    {return this.props.showModalGallery ? this.renderModal() : null}
```

Comment:
I think the `{}` aren't required here, because there is no JSX present, but I'm not 100% sure. What do you think?

Reply:
I think you're right!